README.md

```diff
@@ -50,6 +50,8 @@ The following env variable(s) are optional.
 * `SMTP_NETWORKS` Setting this will allow you to add additional, comma seperated, subnets to use the relay. Used like
     -e SMTP_NETWORKS='xxx.xxx.xxx.xxx/xx,xxx.xxx.xxx.xxx/xx'
 
+* `SMTP_PASSWORD_FILE` Setting this to a mounted file containing the password, to avoid passwords in env variables.
```

Comment:
It should be clear that this variable should point to a file inside the container, an example could help here. It also needs to be added and documented in the .env.example file.

Reply:
added usage to the readme and added var to .env.example